examples/aarch64/rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Can we keep this on stable? It was working with stable before, I'd like to keep it that way.